plugins/kubernetes/test/samples/kubernetes_deployment.yml

```diff
@@ -18,6 +18,9 @@ spec:
       labels:
         project: some-project
         role: some-role
+      annotations:
```

Comment:
Don't strictly need to, but I wanted to make sure that the verifier passed with a file that did contain annotations.